lib/iframe-browser.js

```diff
@@ -0,0 +1,96 @@
+define(function (require, exports, module) {
```

Comment:
You needed to use `git mv` so it sees this as a rename vs. a new file.  Can you fix?